components/published-toggle/index.jsx

```diff
@@ -16,6 +17,20 @@ const PublishedToggle = ({
     else toggletext = 'UnPublished'
   }
 
+  const handleChange = useCallback(async event => {
+    if (disabled) {
+      event.preventDefault()
+      return
+    }
+    const isVisible = event.target.checked
+    try {
+      await onVisibilityChanged(isVisible)
+      toggleChecked(isVisible)
+    } catch (e) {
```

Comment:
Try to use self-explained variables.

```suggestion
    } catch (ignoredNetworkError) {
```